src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableWriter.class.php

```diff
@@ -361,7 +361,7 @@ public function DeleteConnectedRecordReferences()
     {
         parent::DeleteConnectedRecordReferences();
 
-        $this->deleteMultiTableRecordReferencesForDeletedTable($this->oTableConf->sqlData['name']);
+        $this->deleteMultiTableRecordReferencesForDeletedTable($this->_sqlTableName);
```

Comment:
This is now different from "cms_tbl_conf"?

Reply:
Yes. This is the table, that is deleted at this moment.